scripts/macos_installer.py

```diff
@@ -14,14 +14,14 @@ def __init__(self):
             self.THYRA_SERVER_URL = "https://github.com/massalabs/thyra/releases/latest/download/thyra-server_darwin_arm64"
             self.THYRA_APP_URL = "https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_darwin-arm64"
             self.MKCERT_URL = "https://dl.filippo.io/mkcert/latest?for=darwin/arm64"
-            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.9/wallet-plugin_darwin-arm64.zip"
+            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.10/wallet-plugin_darwin-arm64.zip"
             self.THYRA_WALLET_BINARY_FILENAME = "wallet-plugin_darwin-arm64"
             self.THYRA_WALLET_ZIP_FILENAME = "wallet-plugin_darwin-arm64.zip"
         elif platform.machine() == "x86_64":
             self.THYRA_SERVER_URL = "https://github.com/massalabs/thyra/releases/latest/download/thyra-server_darwin_amd64"
             self.THYRA_APP_URL = "https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_darwin-amd64"
             self.MKCERT_URL = "https://dl.filippo.io/mkcert/latest?for=darwin/amd64"
-            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.9/wallet-plugin_darwin-amd64.zip"
+            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.10/wallet-plugin_darwin-amd64.zip"
```

Comment:
This link points to something that does not exists

Reply:
should be latest ? @Thykof  ?

Reply:
to not break thyra install I would like to specify here a fixed version

Reply:
@SlnPons latest or 0.0.11?

Reply:
Latest please :) 
Based on our [release ready definition](https://www.notion.so/massa-innoteam/Plan-for-the-mainnet-c574da44a4854eb3841a5f2e93a2977c?pvs=4#3d75e410a46f4282aad137485ffb3e3b), all releases should be usable. So the latest is the best to start your MassaStation experience.